src/main/java/org/isf/permissions/service/GroupPermissionIoOperations.java

```diff
@@ -37,29 +33,13 @@
 @TranslateOHServiceException
 public class GroupPermissionIoOperations {
 
-	private GroupPermissionIoOperationRepository repository;
+	private final GroupPermissionIoOperationRepository repository;
 
 	public GroupPermissionIoOperations(GroupPermissionIoOperationRepository groupPermissionIoOperationRepository) {
 		this.repository = groupPermissionIoOperationRepository;
 	}
 
-	public List<GroupPermission> findByIdIn(List<Integer> ids) throws OHServiceException {
```

Comment:
You are removing exceptions that can be thrown as you seem to be ignoring the annotation `@TranslateOHServiceException` and how database calls are being wrapped with `org.isf.utils.db.OHServiceExceptionTranslator`.  Please review how the code works and replace the `throws` clauses.
This applies to multiple files.